plugins/inputs/windows_event_log/wineventlog/utils.go

```diff
@@ -168,3 +170,52 @@ func WindowsEventLogLevelName(levelId int32) string {
 		return UNKNOWN
 	}
 }
+
+// In some cases wevtapi does not insert values when formatting the message. The message
```

Comment:
nit: It is good practice in Golang to begin a function header comment with the name of the function.
https://go.dev/blog/godoc
>Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Reply:
Agreed. I don't think we're really writing a Go package that really _needs_ a robust godoc, but it's good either way.

Reply:
Learned it. Will update this.